packages/kit/src/core/preview/index.js

```diff
@@ -86,18 +86,16 @@ export async function preview({
 
 			const parsed = new URL(initial_url, 'http://localhost/');
 
-			const rendered =
-				parsed.pathname.startsWith(config.kit.paths.base) &&
```

Comment:
why remove this check?

Reply:
I honestly don't remember. Without this check, the app is available both at `/` _and_ at the base path, which probably isn't intended. I'll put it back.

Reply:
I just pushed a change to add it back